### PR TITLE
chore/rename-assetportfolio

### PR DESCRIPTION
**Pull request by narve**

### CODE REVIEW

**Comment by narve** [comment]
Any comments on this one? It is not important from our perspective (we still use AssetPortfolio in our API), but it would be nice to get other participants comments/review.

**Comment by narve** [comment]
@cdmNSIDE @ArnaudDebray @lde-nside if no objections, please approve this PR. If any objections, please comment.

File: umei-openapi.json [hunk 1]
```diff
@@ -2225,7 +2225,7 @@
             "nullable": true,
             "readOnly": true
           },
-          "assetPortfolioId": {
+          "PortfolioId": {
```
Comment (cdmNSIDE) [inline umei-openapi.json:2228]: Should it start with a lower case ?

File: umei-openapi.json [hunk 2]
```diff
@@ -2319,7 +2319,7 @@
             "description": "Reference to the market.",
             "nullable": true
           },
-          "assetPortfolioId": {
+          "PortfolioId": {
```
Comment (cdmNSIDE) [inline umei-openapi.json:2322]: Should it start with a lower case ?

File: umei-openapi.json [hunk 3]
```diff
@@ -2450,7 +2450,7 @@
             "type": "string",
             "nullable": true
           },
-          "assetPortfolioId": {
+          "PortfolioId": {
```
Comment (cdmNSIDE) [inline umei-openapi.json:2453]: Should it start with a lower case ?

File: umei-openapi.json [hunk 4]
```diff
@@ -2141,7 +2141,7 @@
         },
         "additionalProperties": true
```
Comment (cdmNSIDE) [inline umei-openapi.json:2142]: Shouldn't we add a description to explain what a portfolio is ? Does it still group several assets ?

File: umei-openapi.json [hunk 5]
```diff
@@ -2319,7 +2319,7 @@
             "description": "Reference to the market.",
             "nullable": true
           },
-          "assetPortfolioId": {
+          "PortfolioId": {
             "type": "string",
             "description": "The asset portfolio for which this order applies.  \r\nRequired on create for sell orders.\r\nNot updateable.",
```
Comment (cdmNSIDE) [inline umei-openapi.json:2324]: Update this description as well ?

**Review by cdmNSIDE — CHANGES_REQUESTED** [review]

**Comment by ArnaudDebray** [comment]
I don't have a strong opinion on this. To me, AssetPortfolio is clear. We can change it to portfolio but I don't see a big added value

**Review by cdmNSIDE — DISMISSED** [review]

**Comment by narve** [comment]
@cdmNSIDE @ArnaudDebray @lde-nside ok to merge this one? I've fixed the conflicts now.

**Review by cdmNSIDE — APPROVED** [review]

**Review by ArnaudDebray — APPROVED** [review]

**Pull request merged**